docs/rst/tutorials/core/deployment/ds_locators_runtime/ds_locators_runtime.rst

```diff
@@ -0,0 +1,369 @@
+
+    .. tab:: Discovery Environment 1
+
+        .. tabs::
+
+            .. tab:: Server_1
+
+                .. code-block:: bash
+
+                    docker run -it --name server_1 ubuntu-vulcanexus:humble-desktop
+
+            .. tab:: Client_talker_1
+
+                .. code-block:: bash
+
+                    docker run -it --name client_pub_1 ubuntu-vulcanexus:humble-desktop
+
+
+
+Configure the discovery entities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The *servers* configuration is really simple, the *server* ``discovery id`` would be set as ``0`` in the first discovery environment, and ``1`` in the other one.
+
+.. tabs::
+
+    .. tab:: Server 0
+
+        .. code-block:: bash
+
+            fastdds discovery --server-id 0
+
+    .. tab:: Server 1
+
+        .. code-block:: bash
+
+            fastdds discovery --server-id 1
+
+The outputs should look similar to the followings:
+
+.. tabs::
+
+    .. tab:: Server 0
+
+        .. code-block:: bash
+
+            ### Server is running ###
+            Participant Type:   SERVER
+            Server ID:          0
+            Server GUID prefix: 44.53.00.5f.45.50.52.4f.53.49.4d.41
+            Server Addresses:   UDPv4:[0.0.0.0]:11811
+
+    .. tab:: Server 1
+
+        .. code-block:: bash
+
+            ### Server is running ###
+            Participant Type:   SERVER
+            Server ID:          1
+            Server GUID prefix: 44.53.01.5f.45.50.52.4f.53.49.4d.41
+            Server Addresses:   UDPv4:[0.0.0.0]:11811
+
+The output displays the ``server id`` set, followed by the server GUID prefix.
+Server addresses ``0.0.0.0`` net mask allows all addresses to be set, but we need each server specific address.
+Finally, the ``11811`` default port would be necessary for further configuration.
+
+In order to set-up the *clients* to discover each other through the *server*, we need to set the *server* data (IP
+address, port and prefix) in all *clients*.
+The port and prefix are already known, and each specific *server* IP address can be obtained introducing the container
+names in the following command:
+
+.. tabs::
+
+    .. tab:: Discovery Environment 0
+
+        .. code-block:: bash
+
+            docker inspect --format '{{ .NetworkSettings.IPAddress }}' server_0
+
+    .. tab:: Discovery Environment 1
+
+        .. code-block:: bash
+
+            docker inspect --format '{{ .NetworkSettings.IPAddress }}' server_1
+
+.. note::
+
+    For this example, these are the used IP addresses for each entity.
+
+    +----------------+------------+
+    | CONTAINER NAME | IP ADDRESS |
+    +================+============+
+    | client_sub     | 172.17.0.2 |
+    +----------------+------------+
+    | client_pub_0   | 172.17.0.3 |
+    +----------------+------------+
+    | client_pub_1   | 172.17.0.4 |
+    +----------------+------------+
+    | server_0       | 172.17.0.5 |
+    +----------------+------------+
+    | server_1       | 172.17.0.6 |
+    +----------------+------------+
```

Comment:
I actually think we can force this within the docker commands so we do not need to make the inspection, although I'm not 100% sure. Docker compose does it, so it should be possible

Reply:
Shall we make a `docker network create --subnet=172.17.0.0/16 vulcanexus_tutorial` previously, and then add the flag `--net vulcanexus_tutorial --ip 172.17.0.X` to each `docker run` shell command?